layers/+tools/transmission/packages.el

```diff
@@ -0,0 +1,64 @@
+;;; packages.el --- Transmission Layer packages File for Spacemacs
+;;
+;; Copyright (c) 2012-2016 Sylvain Benner & Contributors
+;;
+;; Author: Sylvain Benner <sylvain.benner@gmail.com>
+;; URL: https://github.com/syl20bnr/spacemacs
+;;
+;; This file is not part of GNU Emacs.
+;;
+;;; License: GPLv3
+
+(setq transmission-packages
+      '(transmission))
+
+(defun transmission/init-transmission ()
+  (use-package transmission
+    :config (progn
```

Comment:
@d12frosted Thank for clarifying :heart:  
I probably need  `use-package` checklist or a cheat sheet ,I always forget something.